sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -903,6 +906,11 @@ public DiagnosticsProvider getDiagnosticsProvider(CosmosAsyncClient client) {
                     return client.getDiagnosticsProvider();
                 }
 
+                @Override
+                public List<CosmosOperationPolicy> getOperationPolicies(CosmosAsyncClient client) {
+                    return client.requestPolicies;
```

Comment:
Same here - should we make this immutable? or return a copy to prevent changes to the list?

Reply:
Nevermind - it is implicitly addressed - because this is the List returned from CosmosClientBuilder which returns immutable list